modules/acs-secured-cluster-services-overview.adoc

```diff
@@ -24,11 +24,12 @@ In addition, Sensor is responsible for all cluster interactions, such as applyin
 * *Admission controller*: The Admission controller prevents users from creating workloads that violate security policies in {short-title}.
 * *Collector*: Collector analyzes and monitors container activity on cluster nodes.
 It collects container runtime and network activity information and sends the collected data to Sensor.
+* *Scanner V4*: Scanner V4 retrieves and scans images, indexes them, and reports matches to known vulnerabilities in the database. It is the default scanner for {product-title-short} and contains the following components:
```

Comment:
ah just noticed this, too. In the Secured Clusters, there is no vuln matching done. That requires the use of Scanner V4 Matcher installed in the Central cluster. Not sure if we want to make that clear or just remove all references to vulnerabilities here.